### DISCUSSION TITLE
Freeplane and Zotero

### DISCUSSION THREAD

**Discussion by bepolymathe** [freeplane/freeplane#933] [Ideas]
Hello all !

Has anyone ever thought of integrating Freeplane and Zotero ? With the possibilities of annotations of the PDF of Zotero 6 it would be an extraordinary combination for researchers, students or all people working with annotating documentation...

**Comment by euu2021** [comment]
In the old forum (sourceforge) there were a couple of threads about Zotero integration. I don't know if some new possibilities were created with Zotero 6, though.

**Reply by bepolymathe** [reply]
Thanks, i have to search about this. For new zotero possibilities --» https://www.zotero.org/blog/zotero-6/

**Comment by bepolymathe** [comment]
Dear community, 

I would like to inform you about a new addon packaged version of [Freeplane Zotero](https://github.com/petervelosy/freeplane-zotero)  that allows you to import references from your Zotero bibliography entries into a node. Very useful! 

Don't hesitate to contribute to the project if you feel like it!

**Reply by Olifair** [reply]
Just found it and installed. This is really great, thanks a lot, I had been waiting for a long time for this. Being able to integrate Zotero and FreePlane will be so exciting. I will let you know how it goes after having worked a while with this. 
Cheers
Oliver

**Reply by bepolymathe** [reply]
Yes, this is great news. Don't hesitate to share your opinion on the github of the project... and especially in relation with the idea of importing annotations from Zotero ( https://github.com/petervelosy/freeplane-zotero/issues/7 ). All ideas are good to take in this exploratory phase.
